yamllint/plugins.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2020 Satoru SATOH
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+Plugin module utilizing setuptools (pkg_resources) to allow users to add their
+own custom lint rules.
+"""
+try:
+    from pkg_resources import iter_entry_points
+except ImportError:
+    def iter_entry_points(group):
+        """Dummy function for pkg_resources.iter_entry_points."""
+        yield
```

Comment:
Can you explain this `try`/`except`? `iter_entry_points` is not always included in `pkg_resources`?

Reply:
You're absolutely right. I remove that part and push again later.

Reply:
I pushed the commit ae8547f for this.

Reply:
(just a note: I originally thought that there is a case that setuptools is not installed and pkg_resources is missing (thus import should fail) but because yamllint depends on setuptools and such case never happen.)